pysnooper/pysnooper.py

```diff
@@ -23,6 +23,9 @@ def write(s):
         def truncate():
             with open(output, 'w') as output_file:
                 pass
+    elif hasattr(output, "__call__"):
```

Comment:
```suggestion
    elif callable(output):
```

Reply:
Yes I wasn't sure if you wanted to use callable() here as it was briefly removed in earlier py3k versions but was later restored